README.md

```diff
@@ -63,22 +63,35 @@ You can get your dev environment up and running with installing only Docker and
 
 ## Running the docker stack
 
-In the project's root directory, run `make dev_up`. This builds the dev version of the docker images and launches the containers using docker-compose. Stop the running stack with `make dev_down`.
+In the project's root directory, run `make new`. This pulls the dev-tagged "expungeservice" image and launches the containers using docker-compose. Stop the running stack with `make stop`.
 
-After running `make dev_up`, you can navigate to `localhost` in the browser and see the frontend running.  You can now log in using either of the following credentials
+After this target completes, you can navigate to [http://localhost:3000](http://localhost:3000) in the browser and connect to the React dev server with full hot-module reloading. This may take a minute or two to come up before it is available. Check the service with `make frontend_logs`. Once everything is up, you can log in using either of the following credentials:
 
 * Email: admin@email.com, Password: admin
 * Email: user@email.com, Password: user
 
-If you run `docker ps`, you can see the frontend app running on `localhost:3000`, however if you try logging into this instance instead of the frontend at `localhost`, you will get a 500 server error.
+In the course of backend development, one may not need to be running the React/HMR dev server. To build the frontend static files, and use the backend to serve them, run the following command:
```

Comment:
```suggestion
In the course of backend development, one may not need to be running the React/HMR dev server. To build the frontend static files, and use Flask to serve them, run the following command:
```

It took me a minute before I realized it wasn't Nginx you were referring to. I like the fact that we're serving using Flask instead of Nginx for dev.

Reply:
Good suggestion! That will help clarify.